Exercise_1.java

```diff
@@ -7,40 +7,65 @@ class Stack {
   
     boolean isEmpty() 
     { 
-        //Write your code here 
-    } 
+        //Write your code here
+        if(top == 0 ){

```

Comment:
if top == -1 , then it is in empty state . You have initialised your top pointer with -1 . If top ==0 then it means there is still one element in stack.

Reply:
fixed it